core/src/main/scala/kafka/message/ByteBufferMessageSet.scala

```diff
@@ -160,6 +160,10 @@ object ByteBufferMessageSet {
             }
         }
       }
+
+      override def close() = {
```

Comment:
Nitpick, no `=` in this definition (that infers the result type instead of having `Unit` as the return type).